l1-contracts/test/governance/nomismatokopio/mint.t.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.27;
+
+import {Ownable} from "@oz/access/Ownable.sol";
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+import {Errors} from "@aztec/governance/libraries/Errors.sol";
+import {NomismatokopioBase} from "./Base.t.sol";
+
+contract MintTest is NomismatokopioBase {
+  uint256 internal constant RATE = 1e18;
+  uint256 internal maxMint;
+
+  function setUp() public {
+    _deploy(RATE);
+    vm.warp(block.timestamp + 1000);
+
+    maxMint = nom.mintAvailable();
+
+    assertGt(maxMint, 0);
+  }
+
+  function test_GivenCallerIsNotOwner(address _caller) external {
+    // it reverts
+    vm.assume(_caller != address(this));
+    vm.expectRevert(abi.encodeWithSelector(Ownable.OwnableUnauthorizedAccount.selector, _caller));
+    vm.prank(_caller);
+    nom.mint(address(0xdead), 1);
+  }
+
+  modifier givenCallerIsOwner() {
```

Comment:
Nit, comment would be nice that this should be empty because the base contract test deployed it.